packages/headless/src/database/mongodb/wishList/index.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@nestjs/common';
+import { Item, WishList } from 'src/entity/wishList';
+import { IWishListDatabase } from 'src/modules/wishlist/repositories/wishList.database.interface';
+import { WishListModel } from './wishList.model';
+@Injectable()
+export class WishListDatabase implements IWishListDatabase {
+  async findWishListByUserId(userId: string): Promise<WishList | null> {
+    const wishList = await WishListModel.findOne({ user: userId }).lean();
+    return Promise.resolve(wishList);
```

Comment:
Instead of 2 line of code, we can return it on the first line. I've added in my example code. but should be removed

Reply:
done